### PR TITLE
[kubevirt - common-templates]add new centos stream 8 and 9 lanes to common-templates

### PR DESCRIPTION
**Pull request by ksimon1**
add new centos stream 8 and 9 lanes to common-templates
deprecated centos 8 lane

Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/kubevirt/common-templates/kubevirt-common-templates-master-presubmits.yaml [hunk 1]
```diff
@@ -15,8 +15,8 @@ presubmits:
       ci-operator.openshift.io/cloud-cluster-profile: azure4
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-kubevirt-common-templates-master-e2e-azure-common-templates-centos6
-    rerun_command: /test e2e-azure-common-templates-centos6
+    name: pull-ci-kubevirt-common-templates-master-e2e-azure-common-templates-centos-stream8
+    rerun_command: /test e2e-azure-common-templates-centos-stream8
     spec:
```
Comment (dominikholler) [inline ci-operator/jobs/kubevirt/common-templates/kubevirt-common-templates-master-presubmits.yaml:20]: Is this removing test for the centos6 template?

Comment (ksimon1) [reply ci-operator/jobs/kubevirt/common-templates/kubevirt-common-templates-master-presubmits.yaml:20]: No I am not removing centos 6, 7 jobs. I am removing only centos 8 job and adding centos stream 8, 9 jobs. The script which generates these manifests just moved the centos 6, 7 jobs to a bottom (see https://github.com/openshift/release/pull/24986/files#diff-af43041056e5502cbb0dfa0da46c6b109c227d765d96b10e499d64a6ac5d7e5dR224)

Comment (dominikholler) [reply ci-operator/jobs/kubevirt/common-templates/kubevirt-common-templates-master-presubmits.yaml:20]: I got it, thanks.

**Review by dominikholler — APPROVED** [review]
/lgtm

**Pull request merged**